LKExp-Firmware-Disco-Pelion_via_ethernet/mbed_cloud_dev_credentials.c

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) 2018 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __MBED_CLOUD_DEV_CREDENTIALS_H__
+#define __MBED_CLOUD_DEV_CREDENTIALS_H__
+
+#include <inttypes.h>
+
+const char MBED_CLOUD_DEV_BOOTSTRAP_ENDPOINT_NAME[] = "017597e0fd8502ecc0af2a0003c00000";
+const char MBED_CLOUD_DEV_ACCOUNT_ID[] = "017597cf132a560f6f2136c000000000";
+const char MBED_CLOUD_DEV_BOOTSTRAP_SERVER_URI[] = "coaps://bootstrap.us-east-1.mbedcloud.com:5684?aid=017597cf132a560f6f2136c000000000";
```

Comment:
ça ça vient de notre compte mbed?

Reply:
Généré via la commande de device-management, je suppose qu'en partie oui

Reply:
Removed in bee9598